Firmware/MotorControl/motor.cpp

```diff
@@ -177,19 +177,19 @@ bool Motor::do_checks() {
     return true;
 }
 
-float Motor::effective_current_lim() {
+float Motor::effective_torque_lim() {
```

Comment:
Hm actually I think all the limits involved here make more sense to be in terms of current. I would keep this function as effective_current_lim, and apply the conversion at every applicable call site.